submissions/zenzelya/dom-api-task/src/scss/modules/footer.scss

```diff
@@ -0,0 +1 @@
+// footer style
```

Comment:
in some smart books authors wrote that you should never write code in advance
this footer style is redundant